packages/ui/Touchable/Flick.js

```diff
@@ -25,6 +25,17 @@ class Flick {
 		this.move(coords);
 	}
 
+	// This method will get the `onHold`, `onHoldEnd`, `onHoldPulse` props and update the existing `holdConfig` values.
+	updateProps = ({onHold, onHoldEnd, onHoldPulse}) => {
+		// `holdConfig` might not exists a gesture not in progress. Check if gesture exists before updating the references to the `holdConfig`
+		if (!this.holdConfig) return;
+
+		// This will update the `holdConfig` with the new value
+		this.holdConfig.onHoldPulse = onHoldPulse;
+		this.holdConfig.onHold = onHold;
+		this.holdConfig.onHoldEnd = onHoldEnd;
+	}
```

Comment:
This should reference the `Flick`-specific members. In this case, `this.tracking` instead of `this.holdConfig` and `onFlick` instead of `onHold` + others